masterlist.yaml

```
@@ -280,6 +280,10 @@ common:
     display: '[SoundCommands 4.0](https://www.nexusmods.com/oblivion/mods/16288)'
 
   # Mod Files
+  - &Cobl
+    name: 'Cobl Main.esm'
+    display: '[Common Oblivion (Cobl)](https://www.nexusmods.com/oblivion/mods/21104/)'
+
   - &BetterCities
     name: 'Better Cities Resources.esm'
     display: '[Better Cities](https://www.nexusmods.com/oblivion/mods/16513)'
```

```suggestion
  - &BetterCities
    name: 'Better Cities Resources.esm'
    display: '[Better Cities](https://www.nexusmods.com/oblivion/mods/16513)'

  - &Cobl
    name: 'Cobl Main.esm'
    display: '[Common Oblivion (Cobl)](https://www.nexusmods.com/oblivion/mods/21104/)'
```

These should be sorted alphabetically.